### DISCUSSION TITLE
Talent Calculator on website

### DISCUSSION THREAD

**Discussion by hjortmar** [chromiecraft/chromiecraft#3146] [General]
I was curious what abilities and/or talents I get at max lvl but I cannot seem to find anything aboot it.

**Comment by 55Honey** [comment]
Hi hjortmar,
Feel free to check out https://wowgaming.altervista.org/aowow/?talent

**Reply by r-o-b-o-t-o** [reply]
Just remember that you can use only 51 points at the moment 😉
